javascript/apps/taiga/src/app/shared/comments/components/comment-user-input/comment-user-input.component.html

```diff
@@ -0,0 +1,26 @@
+<!--
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+-->
+
+<ng-container *transloco="let t">
+  <ng-container *ngIf="model$ | async as vm">
+    <tg-user-avatar
+      size="m"
+      class="no-border"
+      [user]="vm.user"
+      [color]="vm.user.color"
+      type="light"
+      [rounded]="true"
+      aria-hidden="true"></tg-user-avatar>
+
+    <button
```

Comment:
When the user has 'view only' permissions the button to show the text editor still there
![image](https://github.com/taigaio/taiga/assets/5970959/6ad2a835-0e60-4f1a-ba6b-a08f507e4c61)